releasenotes/acs-emailapi-releasenote.md

```diff
@@ -0,0 +1,25 @@
+# Release notes for Email API
+
+## Email 
+Azure Communication Services new Email APIs to enable application-to-person (A2P) communication. 
+[new SDKs available]( https://docs.microsoft.com/azure/communication-services/concepts/sdk-features) for the following Languages:
+- .NET
+- Javascript
+
+To learn more about [Email](https://docs.microsoft.com/azure/communication-services/concepts/email/email-overview).
+
+## Features Added
+- Simple REST APIs and SDKs.
+- High volume email sending support for A2P (Application to Person) use cases such as appointment reminders, alerts, and notifications for multi-factor authentication, password reset, etc.
+- Custom domain support to enable emails to be sent from email domains verified by your Domain Providers.
```

Comment:
"Domain Providers" does not require capitalization since it's not a named service.